pkg/agent/client.go

```diff
@@ -80,15 +80,16 @@ func newConnectionManager() *connectionManager {
 	}
 }
 
-// AgentIdentifiers stores identifiers that will be used by the server when
+// Identifiers stores agent identifiers that will be used by the server when
 // choosing agents
-type AgentIdentifiers struct {
+type Identifiers struct {
 	IPv4 []string
 	IPv6 []string
 	Host []string
 	CIDR []string
 }
 
+// IdentifierType ...
```

Comment:
Can we use a TODO instead of ... if you want someone to follow up adding verbiage?

Reply:
finally I removed these comments that I added, because this PR is not intended to solve this. Maybe someone (or I) might create another PR for only fixing these verbiage